molecule/default/verify.yml

```diff
@@ -13,3 +14,60 @@
     - name: Check for backup
       stat:
         path: /opt/backup/rsnapshot/hourly.0/backup-defaul/root/rsnapshot/timestamp.hourly
+
+    - name: .my.cnf exists    
+      command: cat /root/.my.cnf
+
+    - name: Create test table in database and insert test value
+      community.mysql.mysql_query:
+        query:
+          - CREATE TABLE IF NOT EXISTS test (message varchar(255) NOT NULL) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+          - INSERT INTO test(message) VALUES('Ansible value');
+        login_db: testDatabase
+
+    - name: Create backup script
```

Comment:
You don't need to run tasks that are available in `tasks/` via `verify.yml`. They will be automatically executed in the `converge` stage.